pace-util/pace/util/global_config.py

```diff
@@ -55,3 +58,57 @@ def is_gtc_backend() -> bool:
 # if FALSE, caches will be checked and rebuild if code changes
 _REBUILD: bool = getenv_bool("FV3_STENCIL_REBUILD_FLAG", "False")
 _VALIDATE_ARGS: bool = True
+
+
+class DaCeOrchestration(enum.Enum):
```

Comment:
Looks like you have it under `dace_config` too, I think if possible let's keep everything there since the only global config usage is dace.

Reply:
Much things were pulled back by this borked merge `-_-` Nothing should be in global_config, I removed it all